liberty-rest-protocol/src/main/java/io/openliberty/arquillian/protocol/rest/runner/SecurityActions.java

```diff
@@ -0,0 +1,337 @@
+/*
+ * JBoss, Home of Professional Open Source
```

Comment:
Why is this copyright different than all the other new files you added? It even has a 2009 year in it when it seems it should be 2022 for a new file.

Reply:
The file is an exact copy of the servlet one, so I left it, but i have updated it now to have `, 2022` added to it.